public/data/gurjeetsinghvirdee.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "Gurjeet Singh Virdee",
+    "type": "personal",
+    "bio": "Self-taught MERN Stack Developer with a strong eye for innovative design and eager to join a creative, problem solving team.",
+    "avatar": "https://github.com/gurjeetsinghvirdee.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/gurjeetsinghvirdee",
+        "icon": "github"
+      },
+      {
+        "name": "Add me on LinkedIn",
+        "url": "https://www.linkedin.com/in/gurjeet-singh-virdee-25a476199/",
+        "icon": "linkedin"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Started Open Source Contribuitions",
+        "date": "Sep 2020",
+        "description": "Started some beginner freind open source projects",
+        "icon": "github",
+        "color": "blue"
+      },
+      {
+        "title": "Started Technical Content Writing",
+        "date": "October 2020",
+        "description": "Writing Technical Articles in GeeksforGeeks",
+        "icon": "book",
+        "color": "red"
+      }
+    ]
+  }
```

Comment:
Missing an endline.
```suggestion
}

```

Reply:
Thankyou for your feedback! I changed the alignment